tests/baselines/reference/user/prettier.log

```diff
@@ -6,6 +6,9 @@ src/cli/util.js(262,64): error TS2339: Property 'length' does not exist on type
 src/cli/util.js(335,52): error TS2339: Property 'length' does not exist on type 'Ignore'.
 src/cli/util.js(396,46): error TS2345: Argument of type 'null' is not assignable to parameter of type 'number | undefined'.
 src/cli/util.js(403,39): error TS2339: Property 'grey' does not exist on type 'typeof import("../../../node_modules/chalk/types/index")'.
+src/cli/util.js(454,16): error TS2339: Property 'type' does not exist on type 'never'.
```

Comment:
a util function `flattenArray` now returns `never[]` instead of `any`. This breaks some downstream code. Here's the code:

```js
function flattenArray(array) {
  return [].concat.apply([], array);
}
```

Reply:
Note that using `Array.prototype.concat.apply` instead makes it return `any[]`, which is the best non-generic type we can infer.